doc/query.md

```diff
@@ -119,6 +195,86 @@ query.put("age", gt12);
 
 See below for supported operators (Selections -> Conditions).
 
+#### Text search
+
+A text clause may be used (if a text index exists) as part of a query to perform full text search term matching, phrase matching, prefix wildcard matching, etc.  A text clause can stand on its own as a query or can be part of a compound query (see below).  Text search supports SQLite Enhanced Query Syntax, see [SQLite full text query][ftsQuery] for more details on syntax that is possible with text search.
```

Comment:
"A text clause may be used (if a text index exists) as part" -> "After creating a text index, a text clause may be used as part"